libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -529,9 +539,17 @@ export class QnAMakerDialog extends WaterfallDialog implements QnAMakerDialogCon
             endpointKey: this.endpointKey.getValue(dc.state),
             host: this.getHost(dc),
         };
-        const options = await this.getQnAMakerOptions(dc);
-        const logPersonalInformation = this.logPersonalInformation && this.logPersonalInformation.getValue(dc.state);
-        return new QnAMaker(endpoint, options, this.telemetryClient, logPersonalInformation);
+
+        const logPersonalInformation = this.logPersonalInformation instanceof BoolExpression ?
+            this.logPersonalInformation.getValue(dc.state)
+            : this.logPersonalInformation
```

Comment:
Needs to allow user to be able to use a regular boolean value instead of only a BoolExpression, since not everybody who uses `QnAMakerDialog` is doing so with adaptive dialogs/DialogManager, which sets the scope of `settings` to pull logpii value

Reply:
Prettier formatting should be adding semicolons.

Reply:
Ran prettier 🌷